src/main/kotlin/io/snyk/plugin/cli/ConsoleCommandRunner.kt

```diff
@@ -63,7 +64,16 @@ open class ConsoleCommandRunner {
         }
 
         logger.debug("Execute ScriptRunnerUtil.getProcessOutput(...)")
-        val processOutput = ScriptRunnerUtil.getProcessOutput(processHandler, ScriptRunnerUtil.STDOUT_OUTPUT_KEY_FILTER, 720000)
+        val timeout = getWaitForResultsTimeout()
+        val processOutput = try {
+            ScriptRunnerUtil.getProcessOutput(
+                processHandler,
+                ScriptRunnerUtil.STDOUT_OUTPUT_KEY_FILTER,
+                timeout
+            )
+        } catch (e: ExecutionException) {
+            "Execution timeout [${timeout / 1000} sec] is reached with NO results produced"
```

Comment:
We should not swallow the exception here. How about using `SentryErrorReporter.captureException(e)`?

Reply:
Why we should not? We intentionally swallow ExecutionException here to show timeout error notification instead of fail with exception.
PS will fix: https://sentry.io/organizations/snyk/issues/2695118718/?project=5956531&referrer=slack

Reply:
Actually have sense to log it in Sentry but not pass it to the user.

Reply:
> Why we should not?
Exception swallowing is an anti-pattern.
We wanna to address issues in case we'll have an increased rate of such exceptions + we will have a full stacktrace.